files/models.py

```diff
@@ -26,8 +24,8 @@ class Inode(FYNode):
     Actual file/dir existing in filesystem
     """
     owner = models.ForeignKey(User)
-    created = models.DateTimeField(default=timezone.now)
-    updated = models.DateTimeField(default=timezone.now)
```

Comment:
Yes.

For now, there's no way `mtime` could possibly be changed after the initial file upload, since:
1. HTTP file upload won't send original file metadata, so we can only set `mtime` to the time it is uploaded
2. there's no Feng Yuan app to change the file content

But `mtime` will be needed in the future. It will be updated in these cases:
1. a client capable of sending original file metadata
2. other server side programs (such as FTP server) may change the file content